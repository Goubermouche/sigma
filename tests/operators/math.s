i32 main() {
    i32 value1 = 100 + 200;
    i32 value2 = 100 - 200;
    i32 value3 = 100 * 200;
    u8  value4 = 32 * 10 - 1; // 63

    i32 precedence = value1 * value2 - value3;

    printf("%d\n", value1);
    printf("%d\n", value2);
    printf("%d\n", value3);
    printf("%d\n", value4);
    printf("%d\n", precedence);
    ret 0;
}
