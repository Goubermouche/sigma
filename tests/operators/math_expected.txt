300
-100
20000
63
-50000
